resteasy-spring-boot-starter/src/main/java/com/paypal/springboot/resteasy/ResteasySpringBootConfig.java

```diff
@@ -35,17 +35,15 @@
 
 	private static Logger logger = LoggerFactory.getLogger(ResteasySpringBootConfig.class);
 
-	private static ResteasyProviderFactory resteasyProviderFactory = new ResteasyProviderFactory();
```

Comment:
@fabiocarvalho777 , have you tried the example at https://github.com/gabrielruiu/resteasy-spring-boot-incorrect-spring-context / http://stackoverflow.com/questions/41909564/resteasy-spring-incorrect-app-instance-loaded-during-tests-with-multiple-runnin ? That explain why you should use a new registry each time.